### PR TITLE
Remove configuration & classes related to lists and their depndencies…

### PR DESCRIPTION
**Pull request by dtvalk-ov**
# Description
Remove configuration & classes related to lists and their dependencies (concepts and concordances)

## Why

[Jira Ticket](https://financialtimes.atlassian.net/browse/UPPSF-2703)

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,6 @@ Document Store API is a Dropwizard application which allows **writes to** and **
 
 The `/content/{uuid}` endpoint is for reading from mongo without any other restriction, no formatting, no hard-coded layout or classes. It's a pure document representation of what is stored in mongoDB.
```
Comment (atanasdinov) [inline README.md:7]: We also support GET `/complementarycontent/{uuid}` and `/internalcomponents/{uuid}` endpoints. Perhaps we could update this as well?

File: runbooks/runbook.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ AWS
 
 ## Architecture
 
-Document Store API allows writes to and reads from MongoDB. Endpoints: `/content` and `/lists`. If this service is unavailable no publishes nor reads from the APIs will be working as expected.
+Document Store API allows writes to and reads from MongoDB. Endpoint: `/content`. If this service is unavailable no publishes nor reads from the APIs will be working as expected.
```
Comment (atanasdinov) [inline runbooks/runbook.md:31]: It's important to keep the runbook up-to-date as much as possible, please add the `/internalcomponents` and `/complementarycontent` endpoints here.

**Review by atanasdinov — COMMENTED** [review]
LGTM. Happy to approve after the following documentation comments are addressed.

File: src/main/java/com/ft/universalpublishing/documentstore/DocumentStoreApiApplication.java [hunk 3]
```diff
@@ -114,8 +91,7 @@ public void run(
 
     environment
         .servlets()
-        .addFilter("cache-filter", new CacheControlFilter("max-age=" + configuration.getCacheTtl()))
-        .addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/lists/*");
+        .addFilter("cache-filter", new CacheControlFilter("max-age=" + configuration.getCacheTtl()));
```
[the remark this answers is not in the document]
Comment (dtvalk-ov) [reply src/main/java/com/ft/universalpublishing/documentstore/DocumentStoreApiApplication.java:94]: Good point. I checked the git history and it seems like the cache filter was added specifically for lists

**Review by atanasdinov — APPROVED** [review]

**Review by Zarrie — APPROVED** [review]

**Pull request merged**